css/src/components/progress-bar.scss

```diff
@@ -70,6 +70,16 @@ $progress-bar-indeterminate-animation-duration: 2s !default;
 		}
 	}
 
+	.theme-dark &.progress-bar-warning {
```

Comment:
`.theme-dark &.*` selectors are final resorts, do we definitely need this? can we instead add something to the theme maps?